Sources/Sentry/SentryFramesTracker.m

```diff
@@ -146,22 +146,18 @@ - (void)displayLinkCallback
     }
 
 #    if SENTRY_TARGET_PROFILING_SUPPORTED
-#        if defined(TEST) || defined(TESTCI)
```

Comment:
l: Is this not meant to run only for tests anymore? Or you forgot to put this back by mistake?

Reply:
No, this was deliberately removed, the change I made now makes it work correctly for tests. Previously, because slicing only took datapoints recorded strictly within the tx timeframe, there would be no frame rate info in a bunch of payloads, so this line made it such that we just had tons of them. It was a hack at the time, glad we could remove it.

Reply:
ah, i might've misremembered. needed this because frame rates are only recorded if the profiler is running, which it is not in the frames tracker tests. i'll try running it for those tests.